src/components/overflow-menu/overflow-menu.js

```diff
@@ -214,33 +241,56 @@ class OverflowMenu extends mixin(createComponent, initComponentBySearch, evented
     const key = event.which;
     const { element, optionMenu, options } = this;
     const isOfMenu = optionMenu && optionMenu.element.contains(event.target);
+    const isExpanded = this.element.getAttribute('aria-expanded') === 'true';
 
-    if (key === 27) {
-      this.changeState('hidden', getLaunchingDetails(event), () => {
-        if (isOfMenu) {
-          element.focus();
+    switch (key) {
+      // Esc
+      case 27:
+        this.element.removeAttribute('aria-expanded');
+        this.changeState('hidden', getLaunchingDetails(event), () => {
+          if (isOfMenu) {
+            element.focus();
+          }
+        });
+        break;
+      // Enter || Space bar
+      case 13:
+      case 32: {
+        if (!isExpanded && document.activeElement !== this.element) {
```

Comment:
```suggestion
        if (!isExpanded && this.element.ownerDocument.activeElement !== this.element) {
```

Background: The codebase avoids global `document`/`window` usage for the sake of scoping, we use `this.element.ownerDocument` for the former, `this.element.ownerDocument.defaultView` for the latter.

Reply:
my mistake, I forgot about this usage